tests/snmpsim/infortrend_infortrend.snmprec

```diff
@@ -0,0 +1,69 @@
+1.3.6.1.2.1.1.1.0|4|Infortrend SNMP agent
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.1714.1.1
+1.3.6.1.2.1.1.3.0|67|130565600
+1.3.6.1.2.1.1.4.0|4|<private>
+1.3.6.1.2.1.1.5.0|4|<private>
+1.3.6.1.2.1.1.6.0|4|<private>
+1.3.6.1.4.1.1714.1.1.2.1.2.1|4|1D57B64E
+1.3.6.1.4.1.1714.1.1.2.1.2.2|4|59FEA16D
+1.3.6.1.4.1.1714.1.1.2.1.3.1|4|0xAEA547A00
+1.3.6.1.4.1.1714.1.1.2.1.3.2|4|0xAEA547A00
+1.3.6.1.4.1.1714.1.1.2.1.4.1|2|17
+1.3.6.1.4.1.1714.1.1.2.1.4.2|2|17
+1.3.6.1.4.1.1714.1.1.2.1.5.1|2|7
+1.3.6.1.4.1.1714.1.1.2.1.5.2|2|6
+1.3.6.1.4.1.1714.1.1.2.1.6.1|2|0
+1.3.6.1.4.1.1714.1.1.2.1.6.2|2|0
+1.3.6.1.4.1.1714.1.1.2.1.7.1|2|0
+1.3.6.1.4.1.1714.1.1.2.1.7.2|2|0
+1.3.6.1.4.1.1714.1.1.2.1.8.1|2|8
+1.3.6.1.4.1.1714.1.1.2.1.8.2|2|7
+1.3.6.1.4.1.1714.1.1.2.1.9.1|2|8
+1.3.6.1.4.1.1714.1.1.2.1.9.2|2|7
+1.3.6.1.4.1.1714.1.1.2.1.10.1|2|1
+1.3.6.1.4.1.1714.1.1.2.1.10.2|2|1
+1.3.6.1.4.1.1714.1.1.2.1.11.1|2|0
+1.3.6.1.4.1.1714.1.1.2.1.11.2|2|0
+1.3.6.1.4.1.1714.1.1.2.1.12.1|4|0
+1.3.6.1.4.1.1714.1.1.2.1.12.2|4|0
+1.3.6.1.4.1.1714.1.1.2.1.13.1|4|0
+1.3.6.1.4.1.1714.1.1.2.1.13.2|4|0
+1.3.6.1.4.1.1714.1.1.3.1.2.1|4|4400FBF43E69FF0B
+1.3.6.1.4.1.1714.1.1.3.1.2.2|4|8E49EBC7D5FE8FC
+1.3.6.1.4.1.1714.1.1.3.1.3.1|4|46881075200
+1.3.6.1.4.1.1714.1.1.3.1.3.2|4|46881075200
+1.3.6.1.4.1.1714.1.1.3.1.4.1|2|21
+1.3.6.1.4.1.1714.1.1.3.1.4.2|2|21
+1.3.6.1.4.1.1714.1.1.3.1.5.1|2|0
+1.3.6.1.4.1.1714.1.1.3.1.5.2|2|0
+1.3.6.1.4.1.1714.1.1.3.1.6.1|2|1
+1.3.6.1.4.1.1714.1.1.3.1.6.2|2|1
+1.3.6.1.4.1.1714.1.1.3.1.7.1|4|1D57B64E 
+1.3.6.1.4.1.1714.1.1.3.1.7.2|4|59FEA16D 
+1.3.6.1.4.1.1714.1.1.4.1.2.1|4|4400FBF43E69FF0B
+1.3.6.1.4.1.1714.1.1.4.1.2.2|4|4400FBF43E69FF0B
+1.3.6.1.4.1.1714.1.1.4.1.2.3|4|4400FBF43E69FF0B
+1.3.6.1.4.1.1714.1.1.4.1.2.4|4|4400FBF43E69FF0B
+1.3.6.1.4.1.1714.1.1.4.1.2.5|4|8E49EBC7D5FE8FC
+1.3.6.1.4.1.1714.1.1.4.1.2.6|4|8E49EBC7D5FE8FC
+1.3.6.1.4.1.1714.1.1.4.1.2.7|4|8E49EBC7D5FE8FC
+1.3.6.1.4.1.1714.1.1.4.1.2.8|4|8E49EBC7D5FE8FC
+1.3.6.1.4.1.1714.1.1.4.1.2.9|4|8E49EBC7D5FE8FC
+1.3.6.1.4.1.1714.1.1.4.1.3.1|4|200000000
+1.3.6.1.4.1.1714.1.1.4.1.3.2|4|000000000
+1.3.6.1.4.1.1714.1.1.4.1.3.3|4|400000000
+1.3.6.1.4.1.1714.1.1.4.1.3.4|4|480000000
+1.3.6.1.4.1.1714.1.1.4.1.3.5|4|004800000
+1.3.6.1.4.1.1714.1.1.4.1.3.6|4|204800000
+1.3.6.1.4.1.1714.1.1.4.1.3.7|4|000000000
+1.3.6.1.4.1.1714.1.1.4.1.3.8|4|484800000
+1.3.6.1.4.1.1714.1.1.4.1.3.9|4|805400000
+1.3.6.1.4.1.1714.1.1.4.1.4.1|4|8589934592
+1.3.6.1.4.1.1714.1.1.4.1.4.2|4|8589934592
+1.3.6.1.4.1.1714.1.1.4.1.4.3|4|2147483648
+1.3.6.1.4.1.1714.1.1.4.1.4.4|4|25794969600
+1.3.6.1.4.1.1714.1.1.4.1.4.5|4|8589934592
+1.3.6.1.4.1.1714.1.1.4.1.4.6|4|10737418240
+1.3.6.1.4.1.1714.1.1.4.1.4.7|4|75497472
+1.3.6.1.4.1.1714.1.1.4.1.4.8|4|15044968448
+1.3.6.1.4.1.1714.1.1.4.1.4.9|4|8589934592
```

Comment:
Hello @ervin09 
Is your SNMPSIM file complete ? Cause it does not include OID 1.3.6.1.4.1.1714.1.1.9.1.1.xxxxx for your sensor ? 
When looking at your sensor definition and the MIB, I have the feeling it won't work. Did you check it in real ?

Reply:
Hi , @PipoCanaja thanks again mate,  well i think it is.  (and yet this is my first device which im adding support to , so please allow some margin of mistakes , lol)  here's an image of what it looks like on my local librenms test machine.  [https://pasteboard.co/gwzx1oIsjme0.jpg](url)

there is also one more thing , i don't know how to solve : on the SNMPSIM file there is a line : luDevDescription.36 = "" value is empty.
libreNMS is sorting the sensors alphabetically leading to this empty description named sensor to be on top... no idea how to avoid that value on yaml.